packages/dx-grid-core/src/plugins/paging-state/computeds.js

```diff
@@ -1,8 +1,16 @@
-export const paginate = (rows, pageSize, page) =>
-  rows.slice(pageSize * page, pageSize * (page + 1));
+const ALL_PAGES_TITLE = 'All';
```

Comment:
The `pageSizeTitle` export is strange, isn't it? It has a localizable string that can't be configured by a user.